app/components/provider_interface/summer_recruitment_banner.rb

```diff
@@ -6,8 +6,12 @@ def render?
 
   private
 
-    def render_body?
-      Time.zone.now < CycleTimetable.apply_2_deadline(RecruitmentCycle.current_year)
+    def time_of_year
+      find_closed? ? 'post_find_closes' : 'pre_find_closes'
+    end
+
+    def find_closed?
+      Time.zone.now > CycleTimetable.find_closes(2021)
```

Comment:
I've hard-coded this to be 2021. This way we won't accidentally revert to something confusing when the cycle moves to the next year